msal/src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -33,8 +33,9 @@
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Core;
 using Microsoft.Identity.Core.Telemetry;
-using System.Threading;
-
+using System.Threading;
+using Microsoft.Identity.Core.Http;
+
 namespace Microsoft.Identity.Client
 {
     /// <summary>
```

Comment:
>/// <summary> [](start = 4, length = 13)

was thinking if we can do inherit docs for both PBA and CCA?

Reply:
Yes, I want to make another documentation cleanup pass.  Added notes to my backlog.